main.go

```diff
@@ -82,14 +85,17 @@ func main() {
 
 	registerRatioController(mgr, conf.MemoryPerCoreLimit, conf.OrganizationLabel)
 
+	// Currently unused, but will be used for the next kyverno replacements
 	psk := &skipper.PrivilegedUserSkipper{
 		Client: mgr.GetClient(),
 
 		PrivilegedUsers:        conf.PrivilegedUsers,
 		PrivilegedGroups:       conf.PrivilegedGroups,
 		PrivilegedClusterRoles: conf.PrivilegedClusterRoles,
 	}
-	registerNodeSelectorValidationWebhooks(mgr, psk, conf)
+	_ = psk
```

Comment:
I left `PrivilegedUserSkipper ` dangling here since we will use it soon when migrating the remaining Kyverno rules.